src/ytdlui/view.clj

```diff
@@ -57,13 +57,19 @@
    [:div
     (for [job jobs]
       [:div.job-container
+       {:id (format "job-%s" (:job_id job))}
        [:div.job-head
         [:div.job-status-icon
          {:style (format "background-color: %s;" (status-color (:status job)))}
          [:img.job-status-img {:title (string/capitalize (:status job))
                                :src (format "/assets/icons/%s.svg" (status-icon (:status job)))}]]
         [:div [:a {:href (:url job)} (or (:title job) (:url job))]]]
        [:div.job-actions
+        (when (#{"done"} (:status job))
+          [:a
+           {:href (format "/#job-%s" (:job_id job))}
+           [:div.job-action
+            [ :span "🔗 Link"]]])
```

Comment:
This clashes quite a bit with the icon style, let's use the `link` one from the [icon set](https://iconoir.com/) where the other icons are also from. Note that the hard-coded color in the SVG should be replaced with `currentColor`.

Reply:
will do!